xfstests/local.config

```diff
@@ -7,6 +7,6 @@ export FUSE_SUBTYP=.passthrough
 export MOUNT_OPTIONS=""
 export TEST_FS_MOUNT_OPTS=""
 
-PASSTHROUGH_PATH=/home/nikratio/libfuse/build/example/passthrough_hp
-SCRATCH_SOURCE=/mnt/src/scratch
-TEST_SOURCE=/mnt/src/test
+export PASSTHROUGH_PATH=<path-to-libfuse-build>/example/passthrough_hp
+export SCRATCH_SOURCE=/mnt/src/scratch
+export TEST_SOURCE=/mnt/src/test
```

Comment:
From your comment in the main discussion thread:

> As written in the new comment, now you can still set TEST/SCRATCH_SOURCE vars,
> but you can instead set TEST_DEV in format source://'source path' instead -
> xfstest will treat them as opaque values, but the mount helper parses the source path to use it.

If that's the case, why not do it here? (remove `SCRATCH_SOURCE` and `TEST_SOURCE` and use `TEST_DEV` and `SCRATCH_DEV` instead). If that works, then I see no reason to keep the workaround variables around.

Reply:
Let's drop the old ways then :-).

Reply:
@bsbernd Could you drop the definitions of `SCRATCH_SOURCE` and `TEST_SOURCE` please?